Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -37,6 +43,19 @@ struct PreplannedListItemView: View {
                 statusView
             }
         }
+        .swipeActions {
+            deleteButton
+        }
```

Comment:
```suggestion
        }
        .contentShape(.rect)
```
This defines the content shape for hit testing. Otherwise the whitespace of the view is not tappable:

<img src="https://github.com/user-attachments/assets/63088057-b0ab-4d80-b7c5-1be59ad966c7" width="600">